### PR TITLE
Bug Fixing ERR_UNSUPPORTED_DIR_IMPORT in web3-eth-abi/lib/esm/coders/encode.js with node >18

### PR DESCRIPTION
**Pull request by kmlcnclk**
https://github.com/web3/web3.js/issues/6535 (Fixing this bug)

### CODE REVIEW

**Comment by luu-alex** [comment]
https://github.com/web3/web3.js/issues/6535#issuecomment-1774216669 include this as well, thanks

**Comment by kmlcnclk** [comment]
> [#6535 (comment)](https://github.com/web3/web3.js/issues/6535#issuecomment-1774216669) include this as well, thanks

I add them. You're welcome.

**Review by jdevcs — APPROVED** [review]
@kmlcnclk also add in changelog , thanks

**Comment by kmlcnclk** [comment]
> @kmlcnclk also add in changelog , thanks

I did

File: CHANGELOG.md [hunk 1]
```diff
@@ -1460,6 +1460,10 @@ should use 4.0.1-alpha.0 for testing.
 
 -   Bug fix of `checkNetwork` in ENS (#5988)
 
+#### web3-eth-abi
+
+-   Bug fix of `ERR_UNSUPPORTED_DIR_IMPORT` in ABI (#6535)
+
```
Comment (jdevcs) [inline CHANGELOG.md]: @kmlcnclk not here, under https://github.com/web3/web3.js/blob/4.x/packages/web3/CHANGELOG.md#unreleased and https://github.com/web3/web3.js/blob/4.x/packages/web3-eth-abi/CHANGELOG.md#unreleased , so I merge this PR. Thanks

Comment (kmlcnclk) [reply CHANGELOG.md]: I am so sorry. I commit it now. Is it ok?

**Review by avkos — APPROVED** [review]

**Pull request merged**